server/blob/blob-s3/src/test/java/org/apache/james/blob/objectstorage/aws/S3MinioTest.java

```diff
@@ -120,4 +121,9 @@ void saveShouldWorkForAllPayloadsWithHash() {
             .block();
     }
 
+    @Disabled("Minio changed and returns a generic S3Exception instead of a NoSuchBucketException in its latest release")
```

Comment:
Please better describe the impact on existing features: does this impact some features used by us? By our customers? 

Let's document MINIO limitations on the S3 config page on the Antora website maybe?

Reply:
I dont think customer mentions should figure on an apache open source project comments?

But I will enhance that a bit. Difference is if you try to delete a bucket via our API with MinIO, instead of it being silenced if the bucket does not exist (making it transparent to the user), James will return an error.

Reply:
=> https://github.com/apache/james-project/pull/1675

Separated PR, as suggested by @quantranhong1999 , as this issue is a blocker on master branch but we seem to have something else suspicious on the build with other changes. 

But it addresses your concern.